CONTRIBUTING.md

```diff
@@ -25,6 +25,8 @@
 
    ***
 
+   > Tip: if you have nix installed, you can run `nix develop` to take advantage of the projects dev-shell flake. This will provide any needed tools and dependencies in a reproducible way.
```

Comment:
```suggestion
   > **Note** if you have nix installed, you can run `nix develop` to take advantage of the projects dev-shell flake. This will provide any needed tools and dependencies in a reproducible way.
```